jenkins_callback.py

```diff
@@ -26,7 +26,7 @@ def build_jobs(jenkins_url, jobs_data, headers = {}, user="cmssdt"):
              "Submit": "Build"
            }
     try:
-      data = urlencode(data)
+      data = urlencode(data).encode()
```

Comment:
@iarspider , does this work for both python2 and python3. This code is used by cmssdt-web too ( https://github.com/search?q=repo%3Acms-sw%2Fcmssdt-web%20jenkins_callback&type=code )

Reply:
yes looks good